pkg/api/validation/validation_test.go

```diff
@@ -1742,7 +1742,7 @@ func TestValidateService(t *testing.T) {
 			name: "valid external load balancer 2 ports",
 			tweakSvc: func(s *api.Service) {
 				s.Spec.CreateExternalLoadBalancer = true
-				s.Spec.Ports = append(s.Spec.Ports, api.ServicePort{Name: "p", Port: 12345, Protocol: "TCP"})
+				s.Spec.Ports = append(s.Spec.Ports, api.ServicePort{Name: "velb2p", Port: 12345, Protocol: "TCP"})
```

Comment:
The s.spec.Ports have two ServicePort instances, one is defined in the makeValidService() function, the name of the first one is also "p", so I changed the name of the second one.